curriculum/challenges/english/25-front-end-development/lab-availability-table/66b36358ed4f261d64840c24.md

```diff
@@ -82,10 +82,10 @@ assert.isAtLeast(document.querySelectorAll('tr[class~="half"]').length, 2);
 
```

Comment:
Could you fix the typo in the hint above? I cannot make the suggestion (thanks GitHub).
It should be :
````
`th` elements in the first column should have the `class` of `time` and contain time values.
````

Reply:
Sure thing.